### PR TITLE
Automate census docker build

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
We want travis build to push docker images to Google Container Registry instead of dockerhub

# What has changed
Replaced docker image prefix with GCR registry location
NOTE: before merge, remove "automate-census-docker-build" branch condition and "branches only" section in .travis.yml file

# How to test?
From travis dashboard, switch to  "automate-census-docker-build" branch and "Restart Build". Once complete, check latest Docker image is in GCR.

# Links

[Trello](https://trello.com/c/dC6sEWqE/443-set-up-automated-docker-image-build)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,23 +19,17 @@ install: mvn install -DskipTests -DskipITs -Ddocker.skip -Ddockerfile.skip -Dmav
...
-      secure: xHvqsIklJ+gX67JbpTdi0TMHxMKV9llXH6htk8qIe/sRweq/8tgcTFhSnp6Zlw42cFH6jeweme8xXH6xS3Cft6gLj8nHwAzpkU6cgDndUDPiN1pgCZOBHmIccFcWrl81REfJVgyOVicyKgVthdV1VLCDUkj9Cq+67LKq+EUi9EUw5/1ZzAkPDmuhzinelZ50lu27jyWcPQkKjN4XnpLYB2ybyqqBAJkKaxwooxCZifq1Ghub7a/EH7wiCZgc0zyJfg6EvY4MkJgbfj6b449PRTYaqNxj8Cpy0hO7EsALSednjbjoJtH3+axAFoHdVvskj0QAc3Q9pZfddXWajK4uvWOxf8EifvbhOX+yMJDy7olfuDfVD+BuCSwUXKycbJNA5C4A9fSvj/aoHI/kxxm+/ydToSaZVxLmIa5WR8EU215AAzP3RnOfYDcpPAbYLtq+TdqMuxmns+4mNjOflu4EaRexlNgWbnEwRDvuvgoS/ALEFr2fHrC/LSoMDdgb0pxDE7ddFmAsutWT8zbhGjrNF3mXlfJWEu4xky3rmUsc3GiB5xebPnPaagrVciwHkyJenn8zywBn+zk3u/VfQuHTN9u04dwf38FNC/QZBNj9AFy9wyn9YxWVvsS7euEkpbMLorz1fcIQL+H1sP8C3tZqOleurY+0mkJRHMsiGSjwwx4=
-
 cache:
   directories:
   - $HOME/.m2
 
 branches:
   only:
-  - master
+    - automate-census-docker-build
```
Comment (davidmort) [inline .travis.yml]: shouldn't this be "rm-census"?

File: .travis.yml [hunk 2]
```diff
@@ -14,23 +19,17 @@ install: mvn install -DskipTests -DskipITs -Ddocker.skip -Ddockerfile.skip -Dmav
 script: mvn verify cobertura:cobertura-integration-test
 
 after_success:
-- if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
-  docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-  docker push sdcplatform/collectionexercisesvc;
+- if ([ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || ([ "$TRAVIS_BRANCH" == "automate-census-docker-build" ]); then
```
Comment (davidmort) [inline .travis.yml]: need to remove the second part of this "if" as per the PR description before it can be merged

File: .travis.yml [hunk 3]
```diff
@@ -6,6 +6,11 @@ services:
 language: java
 jdk: openjdk8
 
+env:
+  global:
+    - SOURCE_IMAGE_NAME="collectionexercisesvc"
```
Comment (davidmort) [inline .travis.yml]: Is there any benefit the env vars given they're only used once?

File: .travis.yml (hunk 1 above)
Comment (AdeHarrison) [reply .travis.yml]: fixed

File: .travis.yml (hunk 2 above)
Comment (AdeHarrison) [reply .travis.yml]: fixed

**Pull request merged**